### PR TITLE
twitter show <username> #564

### PR DESCRIPTION
**Pull request by dev-hartmann**

### CODE REVIEW

File: src/yetibot/commands/twitter.clj [hunk 1]
```diff
@@ -63,6 +63,13 @@
       (format "Stopped tracking %s" topic))
     (format "You're not tracking %s" topic)))
 
+(defn show
+  "twitter show <screen-name> # show top 10 tweets from user <scree-name>"
+  [{[_ screen-name] :match}]
+  (let [tweets (:body (model/user-timeline screen-name 10))]
+    (into [] (map #(model/format-tweet-text %) tweets ))))
```
Comment (devth) [inline src/yetibot/commands/twitter.clj]: 1. You can simplify `(map #(model/format-tweet-text %) tweets)` to be `(map model/format-tweet-text tweets)`
1. Do you need the `into []` here? The output of `map` will already be a collection so you should be able to omit

**Comment by devth** [comment]
This looks great! Just had a few minor comments.

**Comment by dev-hartmann** [comment]
Hi, you are totally right,  I forgot to take it out after experimenting. Thanks for the map hint, makes perfect sense. Still feeling like a complete newbie ;) 

I updated the PR.

**Comment by devth** [comment]
@dev-hartmann awesome, thanks for contributing! 👍

**Comment by dev-hartmann** [comment]
@devth You're welcome, thanks for supporting me!

**Pull request merged**